test/jdk/java/io/BufferedReader/ReadAll.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 8354724
+ * @summary Test BufferedReader readAllLines and readString methods
+ * @library .. /test/lib
+ * @build jdk.test.lib.Platform jdk.test.lib.RandomFactory
+ * @run junit ReadAll
+ * @key randomness
+ */
+
+import java.io.BufferedReader;
+import java.io.File;
```

Comment:
Cosmetic thought - I think this could all be done with Path, avoiding File and friends.

Reply:
possibly even using Reader.of(CharSequence)

Reply:
Test modified in b5d8266.